tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Features/Organisations Healthcheck.3538.feature

```diff
@@ -0,0 +1,16 @@
+﻿Feature: Buying Catalogue Organisations healthchecks
+	As BuyingCatalogueOrganisations
+	I want to be be able to check the health of my dependencies
+    So that I can behave accordingly
```

Comment:
Spacing/alignment – it looks like there's a mix of tabs and spaces here. @CallumBradburyBJSS had a similar issue. I can't remember whether we want all tabs or all spaces – @CallumBradburyBJSS can you confirm which it should be?

Reply:
all spaces, the mix of tabs and spaces keeps popping up

Reply:
👍